packages/core/README.md

```diff
@@ -1,6 +1,170 @@
+                        },
+                        "traceabilityInfo": {
+                          "type": "array",
+                          "title": "Traceability Information",
+                          "items": {
+                            "type": "object",
+                            "properties": {
+                              "EventReference": {
+                                "type": "string",
+                                "title": "Event Reference"
+                              },
+                              "EventType": {
+                                "type": "string",
+                                "title": "Event Type"
+                              }
+                            }
+                          }
+                        },
+                    },
+                    "sustainabilityScore": {
+                      "type": "integer",
+                      "title": "Sustainability Score"
+                    },
+                    "trustScore": {
+                      "type": "integer",
+                      "title": "Trust Score"
+                    }
+                  }
+                },
+                "initialData": {},
+                "className": "json-form",
+                "style": { "margin": "auto", "paddingTop": "40px", "width": "80%" }
+              }
+            },
+            {
+              "name": "CustomButton",
+              "type": "Submit",
+              "props": {
+                "style": { "textAlign": "center", "paddingTop": "20px" }
+              }
+            }
+          ],
+          "services": [
+            {
+              "name": "processObjectEvent",
+              "parameters": [
+                {
+                  "vckit": {
+                    "vckitAPIUrl": "https://vckit.example.com",
+                    "issuer": "did:web:issuer.example.com",
+                  },
+                  "dpp": {
+                    "context": ["https://www.w3.org/2018/credentials/v1"],
+                    "renderTemplate": [
+                      { "template": "<p>Render dpp template</p>", "@type": "WebRenderingTemplate2022" }
+                    ],
+                    "type": ["DigitalProductPassport"],
+                    "dlrLinkTitle": "Livestock Passport",
+                    "dlrIdentificationKeyType": "nlisid",
+                    "dlrVerificationPage": "http://example.com/verify"
+                  },
+                  "dlr": {
+                    "dlrAPIUrl": "http://dlr.example.com",
+                    "dlrAPIKey": "5555555555555"
+                  },
+                  "storage": {
+                    "storageAPIUrl": "https://storage.example.com",
+                    "bucket": "example-bucket"
+                  },
+                  "identifierKeyPaths": ["herd", "NLIS"]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
```

Comment:
I think we should use a markdown relative link for this file instead of displaying the raw file here. You can refer to this link if you prefer: [GitHub relative link in markdown file](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file).